easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.20-foss-2015b.eb

```diff
@@ -0,0 +1,29 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Robert Schmidt <roschmidt@ohri.ca>, Cedric Laczny <cedric.laczny@uni.lu>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+name = 'SAMtools'
+version = '0.1.20'
+
+homepage = 'http://www.htslib.org/'
+description = """SAM Tools provide various utilities for manipulating alignments in the SAM format,
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'foss', 'version': '2015b'}
+toolchainopts = {'optarch': True, 'pic': True}
```

Comment:
`optarch` is enabled by default, no need to hardcode it here

existing `SAMtools` easyconfig don't have `pic` enabled, should we do it for all of them? (not in this PR please)